<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ELoad_InputHomeWid</class>
 <widget class="QWidget" name="ELoad_InputHomeWid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>576</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox{
border:2px solid rgb(15,80,232);
border-radius:15px
}</string>
     </property>
     <property name="title">
      <string>电子可控负载一</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <item row="0" column="0">
       <widget class="QWidget" name="widget_1" native="true"/>
      </item>
      <item row="0" column="1">
       <widget class="QWidget" name="widget_2" native="true"/>
      </item>
      <item row="0" column="2">
       <widget class="QWidget" name="widget_3" native="true"/>
      </item>
      <item row="0" column="3">
       <widget class="QWidget" name="widget_4" native="true"/>
      </item>
      <item row="1" column="0">
       <widget class="QWidget" name="widget_5" native="true"/>
      </item>
      <item row="1" column="1">
       <widget class="QWidget" name="widget_6" native="true"/>
      </item>
      <item row="1" column="2">
       <widget class="QWidget" name="widget_7" native="true"/>
      </item>
      <item row="1" column="3">
       <widget class="QWidget" name="widget_8" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox{
border:2px solid rgb(15,80,232);
border-radius:15px
}</string>
     </property>
     <property name="title">
      <string>电子可控负载二</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <item row="0" column="0">
       <widget class="QWidget" name="widget_9" native="true"/>
      </item>
      <item row="0" column="1">
       <widget class="QWidget" name="widget_10" native="true"/>
      </item>
      <item row="0" column="2">
       <widget class="QWidget" name="widget_11" native="true"/>
      </item>
      <item row="0" column="3">
       <widget class="QWidget" name="widget_12" native="true"/>
      </item>
      <item row="1" column="0">
       <widget class="QWidget" name="widget_13" native="true"/>
      </item>
      <item row="1" column="1">
       <widget class="QWidget" name="widget_14" native="true"/>
      </item>
      <item row="1" column="2">
       <widget class="QWidget" name="widget_15" native="true"/>
      </item>
      <item row="1" column="3">
       <widget class="QWidget" name="widget_16" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox{
border:2px solid rgb(15,80,232);
border-radius:15px
}</string>
     </property>
     <property name="title">
      <string>电子可控负载三</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <item row="0" column="0">
       <widget class="QWidget" name="widget_17" native="true"/>
      </item>
      <item row="0" column="1">
       <widget class="QWidget" name="widget_18" native="true"/>
      </item>
      <item row="0" column="2">
       <widget class="QWidget" name="widget_19" native="true"/>
      </item>
      <item row="0" column="3">
       <widget class="QWidget" name="widget_20" native="true"/>
      </item>
      <item row="1" column="0">
       <widget class="QWidget" name="widget_21" native="true"/>
      </item>
      <item row="1" column="1">
       <widget class="QWidget" name="widget_22" native="true"/>
      </item>
      <item row="1" column="2">
       <widget class="QWidget" name="widget_23" native="true"/>
      </item>
      <item row="1" column="3">
       <widget class="QWidget" name="widget_24" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
